typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -354,6 +354,12 @@ corresponding to @racket[trest], where @racket[bound]
 
 @ex[(box "hello world")]
 
+@defform[(Read-Only-Boxof t)]{A read-only @rtech{box} of @racket[t].}
+
```

Comment:
C has a `const` keyword, that you can put on variables that essentially means "I will not modify that" (that also does not imply immutability). That sounds similar, and maybe pointing out that similarity will help people familiar with C understand TR's read-only boxes.